postgres/initdb.sql

```diff
@@ -223,10 +200,25 @@ BEGIN
         'Tried to insert invalid result: Project: % Group: % Task: % - User: %', v.project_id, v.group_id, NEW.task_id, v.user_id
             USING ERRCODE = '23503';
     END IF;
-
     RETURN NEW;
 END;
 $$;
 
 CREATE TRIGGER insert_mapping_sessions_results BEFORE INSERT ON mapping_sessions_results
     FOR EACH ROW EXECUTE PROCEDURE mapping_sessions_results_constraint();
+
+-- Used to group results by user groups
+CREATE TABLE IF NOT EXISTS mapping_sessions_user_groups (
+    mapping_session_id int8,
+    user_group_id varchar,  -- user group primary key
+    PRIMARY KEY (mapping_session_id, user_group_id),
+    FOREIGN KEY (mapping_session_id) REFERENCES mapping_sessions (mapping_session_id)
+);
+
+-- create table for user_group_results import through csv
+CREATE TABLE IF NOT EXISTS results_user_groups_temp (
```

Comment:
to be consistent should this be named to `mapping_sessions_user_groups_temp` then as well?

Reply:
I think `results_user_groups_temp` is okay for now as it stores results data instead of mappin_sessions. Which is project_id, group_id and user_group_id instead of mapping_session_id.